x/fbridge/keeper/auth_test.go

```diff
@@ -52,10 +52,6 @@ func TestAssignRole(t *testing.T) {
 	err = k.updateRole(ctx, types.RoleOperator, addrs[1])
 	require.NoError(t, err)
 	require.Equal(t, types.RoleMetadata{Guardian: 1, Operator: 1, Judge: 0}, k.GetRoleMetadata(ctx))
-
```

Comment:
Or, by confirming it does not return an error, we can recycle the obsolete test logic into a nice verification on the role update specification (i.e. it will silently ignore the suggestion, making it idempotent).